tools/downloader/topologies_test.py

```diff
@@ -0,0 +1,68 @@
+import unittest
+import os
+import numpy as np
+import cv2 as cv
+
+import topologies as omz
+
+from openvino.inference_engine import IECore
+
+class TestTopologies(unittest.TestCase):
+
+    download_path = os.environ.get("OPEN_MODEL_ZOO_CACHE_DIR")
+    ssd = omz.load_model('mobilenet-ssd')
+
+    def test_load_model(self):
+        self.assertTrue(os.path.exists(
+            os.path.join(self.download_path, 'public', 'mobilenet-ssd', 'mobilenet-ssd.caffemodel')))
+
+    def test_get_ir(self):
+        xml_path, bin_path = self.ssd.get_ir()
+        self.assertTrue(os.path.exists(xml_path))
+        self.assertTrue(os.path.exists(bin_path))
+        self.assertEqual(os.path.basename(xml_path), 'mobilenet-ssd.xml')
+        self.assertEqual(os.path.basename(bin_path), 'mobilenet-ssd.bin')
+
+    def test_result_model(self):
+        image_path = os.path.join(os.environ.get("OPEN_MODEL_ZOO_PATH"),
```

Comment:
Let's avoid this variable - you may get script's location by `__file__` and find test image by relative path